EasyPost/Services/AddressService.cs

```diff
@@ -34,10 +35,22 @@ internal AddressService(EasyPostClient client) : base(client)
         ///     * {"strict_verifications", List&lt;string&gt;} Possible items are "delivery" and "zip4".
         ///     All invalid keys will be ignored.
         /// </param>
+        /// <param name="verify">Whether to verify this address during the creation process. Default: false</param>
+        /// <param name="verifyStrict">Whether to strictly verify this address during the creation process. Default: false</param>
         /// <returns>EasyPost.Address instance.</returns>
         [CrudOperations.Create]
-        public async Task<Address> Create(Dictionary<string, object> parameters)
+        public async Task<Address> Create(Dictionary<string, object> parameters, bool verify = false, bool verifyStrict = false)
```

Comment:
Please rollback these changes. As with other items, we were going to focus on updated params at a later date across all libs. Let's not do that now

Reply:
These are needed because "verify" and "verify_strict" are not wrapped with the rest of the parameters in the "address" key.

Reply:
The problem is we have a usage pattern established for years now where these are passed in the dictionary. We should handle something like this with a major release and call it out and instead fix this under the hood since it's what users have expected for years and this major release already went out (AKA: this is a bug we didn't catch in time and shouldn't be a feature change)

Reply:
Alright, resolved, but it seems we agree this should be properly addressed in the future.